packages/browserify/test/util.js

```diff
@@ -397,7 +397,7 @@ async function getStreamResults (stream) {
   return results
 }
 
-async function runBrowserify ({ projectDir, scenario }) {
+async function runBrowserify ({ scenario }) {
```

Comment:
what motivated you to put the dir on the scenario

Reply:
One less thing to pass in, more contained. Let me know if you disagree